### ISSUE TITLE
deployment error

### ISSUE THREAD

**Issue by lakshmia77** [mehmetgoren/feniks#38]
hii i want to run the code manually and how to `deploy` after some logo change

**Comment by Petrox** [comment]
```
cd feniks.web_app ;       docker build -t gokalpgoren/webapp:v1.0.3-custom . ;cd ..
cd ffmpeg_service ;         docker build -t gokalpgoren/ffmpeg_service:v1.0.2-custom -f Dockerfile.raspi4 . ;cd ..
cd mngr ;         docker build -t gokalpgoren/mngr:v1.0.3-custom . ;cd ..
cd senseai_service ;         docker build -t gokalpgoren/senseai_service:v1.0.3-custom -f Dockerfile.jetson . ;cd ..
cd smcp ;         docker build -t gokalpgoren/smcp:v1.0.3-custom . ;cd ..
cd snapshot_service ; docker build -t gokalpgoren/snapshot_service:v1.0.3-custom . ; cd ..
````

**Comment by Mo7mud** [comment]
> ```
> cd feniks.web_app ;       docker build -t gokalpgoren/webapp:v1.0.3-custom . ;cd ..
> cd ffmpeg_service ;         docker build -t gokalpgoren/ffmpeg_service:v1.0.2-custom -f Dockerfile.raspi4 . ;cd ..
> cd mngr ;         docker build -t gokalpgoren/mngr:v1.0.3-custom . ;cd ..
> cd senseai_service ;         docker build -t gokalpgoren/senseai_service:v1.0.3-custom -f Dockerfile.jetson . ;cd ..
> cd smcp ;         docker build -t gokalpgoren/smcp:v1.0.3-custom . ;cd ..
> cd snapshot_service ; docker build -t gokalpgoren/snapshot_service:v1.0.3-custom . ; cd ..
> ```

Can we run those services on Linux without docker?
If yes, how?

**Comment by Petrox** [comment]
> Can we run those services on Linux without docker? If yes, how?

It should be possible, but you'd need to look inside the Dockerfile and follow each command on your PC, which should have the same environment and software versions at the beginning as the base docker image referred in the first line.
Given Feniks has more than one service (at least 6 or more in the 6 containers) you'd need to install all prerequisites then build and configure everything on your baremetal. Plus you need to set up permissions, storage folders and networking of course.

TLDR: you either go for the container way, or configure your system with manual labor and then it will be brittle, a simple apt-get upgrade ffmpeg could break everything if the components are tightly integrated and dependent on unstable apis.
